app/tests/integration/nimbus/test_desktop_client_integrations.py

```diff
@@ -190,3 +197,179 @@ def test_check_telemetry_enrollment_unenrollment(
     else:
         if control is not False:
             assert False, "Experiment unenrollment was never seen in Ping Data"
+
+
+@pytest.mark.nimbus_integration
+@pytest.mark.xdist_group(name="group2")
+def test_check_telemetry_enrollment_unenrollment_pref_flip(
+    base_url,
+    selenium,
+    kinto_client,
+    slugify,
+    experiment_name,
+    create_desktop_experiment,
+    trigger_experiment_loader,
+):
+    _row_locator = (By.CSS_SELECTOR, "tr > td > span > span")
+    _search_bar_locator = (By.ID, "about-config-search")
+
+    requests.delete("http://ping-server:5000/pings")
+    targeting = helpers.load_targeting_configs()[0]
+    experiment_slug = str(slugify(experiment_name))
+    data = {
+        "hypothesis": "Test Hypothesis",
+        "application": "DESKTOP",
+        "changelogMessage": "test updates",
+        "targetingConfigSlug": targeting,
+        "publicDescription": "Some sort of Fancy Words",
+        "riskRevenue": False,
+        "riskPartnerRelated": False,
+        "riskBrand": False,
+        "featureConfigId": 9,
+        "referenceBranch": {
+            "description": "reference branch",
+            "name": "Branch 1",
+            "ratio": 99,
+            "featureEnabled": True,
+            "featureValue": '{"value": "test_string_automation"}',
+        },
+        "treatmentBranches": [
+            {
+                "description": "treatment branch",
+                "name": "Branch 2",
+                "ratio": 1,
+                "featureEnabled": False,
+                "featureValue": "",
+            }
+        ],
+        "populationPercent": "100",
+        "totalEnrolledClients": 55,
+    }
+    create_desktop_experiment(
+        experiment_slug,
+        "desktop",
+        targeting,
+        data,
+    )
+    wait = WebDriverWait(selenium, 10)
+    selenium.get("about:config")
+    search_bar = wait.until(EC.presence_of_element_located(_search_bar_locator))
+    search_bar.send_keys("nimbus.qa.pref-1")
+    wait.until(EC.presence_of_element_located(_row_locator))
+    elements = selenium.find_elements(*_row_locator)
+    for item in elements:
+        if "default" in item.text:
+            assert True
+        else:
+            continue
```

Comment:
Could wrap this in one of those handy page objects so it's a nice clean interface like the other pages?